### PR TITLE
ListItem: always show Forget button; add Buy button conditionally

### PR DESCRIPTION
**Pull request by dmytrokulak**
1) always show Forget button 
 2) lock Buy button until time has lapsed and some awareness questions answered
 3) add emoticons in alerts 
4) replace icons in Forget button and Questions button

### CODE REVIEW

File: src/Components/PageSelection/PageSelection.js [hunk 1]
```diff
@@ -14,15 +14,19 @@ const PageSelection = (props) => {
...
-        <Home
-          allProducts={props.allProducts}
-          changePage={setSelectedPage}
-          setStaticHeader={props.setStaticHeader}
-          updateProduct={props.updateProduct}
-          removeProduct={props.removeProduct}
-          setCurrentProduct={props.setCurrentProduct}
-          increaseLikeCount={props.increaseLikeCount}
-        />
+        <Fragment>
```
Comment (brurez) [inline src/Components/PageSelection/PageSelection.js:17]: We would use `Fragment` here if we have multiple elements. Just a side note, changes are not necessary.

Comment (dmytrokulak) [reply src/Components/PageSelection/PageSelection.js:17]: Right, I used to have a couple of things here at some point in development and missed to remove the Fragment in the end.

Comment (dmytrokulak) [reply src/Components/PageSelection/PageSelection.js]: Thanks
f719f38

**Review by brurez — CHANGES_REQUESTED** [review]
image
I didn't get the meaning of the buttons when the user has not yet answered 3 questions. In this case, we could replace the label "no" with "close". Also, we can remove the "yes" button in this case.

File: src/Components/Home/ProductRemovedAlert.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const ProductRemovedAlert = (props) => {
+  return (
+    <Fragment>
+      <Alert show={props.showForgetAlert} variant="success" className="my-1">
+        <p>
+          <i className="far fa-smile"></i> The product has been removed from the
+          list. Good job!
+        </p>
+      </Alert>
+      <Alert show={props.showBoughtAlert} variant="danger" className="my-1">
```
Comment (magg01) [inline src/Components/Home/ProductRemovedAlert.js:13]: Could we change the alert when a product is bought after the waiting period has elapsed?

I think the current Alert of red colour and frown makes sense for if the User skipped the waiting period. 

However I think waiting 30 days, answering the consideration questions and then deciding to buy the product should be a success event in the app with a more encouraging message. Perhaps a message along the lines of "Congratulations on considering this purchase carefully. Go ahead without regrets."

**Review by magg01 — CHANGES_REQUESTED** [review]

File: src/Components/Home/ProductRemovedAlert.js (hunk 2 above)
Comment (dmytrokulak) [reply src/Components/Home/ProductRemovedAlert.js:13]: we don't track if a user skipped a waiting period ... 😕

File: src/Components/Home/ListItem.js [hunk 3]
```diff
@@ -1,59 +1,74 @@
...
+          <Col xs={{ span: 8 }} className="text-right">
+            <Button
+              variant="outline-danger"
+              title="Buy product"
+              className="px-3 mb-1 mx-1"
+              onClick={() => setShowBuyDialog(true)}
+              disabled={!canBuy}
+            >
+              <i className="fas fa-shopping-cart"></i>{" "}
+              {canBuy ? "Buy" : daysLeft + " days to unlock"}
```
Comment (magg01) [inline src/Components/Home/ListItem.js:42]: for countdown message on the button, how about `daysLeft + " days to consider"` ?

File: src/Components/Home/ProductRemovedAlert.js (hunk 2 above)
Comment (dmytrokulak) [reply src/Components/Home/ProductRemovedAlert.js:13]: This comment can be a case for further research with users.

File: src/Components/Home/ListItem.js (hunk 3 above)
Comment (dmytrokulak) [reply src/Components/Home/ListItem.js:42]: This comment can be a case for further research with users.

File: src/Components/Home/ProductRemovedAlert.js (hunk 2 above)
Comment (magg01) [reply src/Components/Home/ProductRemovedAlert.js:13]: > we don't track if a user skipped a waiting period ... 😕

not yet, but I feel there should be an option for a user to do that in a later iteration

Comment (dmytrokulak) [reply src/Components/Home/ProductRemovedAlert.js:13]: Such functionality is called "auditing" and it's hell of a work. I'm not convinced. We may have include this in user testing after Milestone 3, or even later.

**Pull request merged**